include/xsimd/arch/xsimd_neon.hpp

```diff
@@ -1577,6 +1577,44 @@ namespace xsimd
             return vget_lane_f32(tmp, 0);
         }
 
+        /**************
+         * reduce_max *
+         **************/
+
+        WRAP_UNARY_INT(vmaxvq, detail::identity_return_type)
+        WRAP_UNARY_FLOAT(vmaxvq, detail::identity_return_type)
+
+        template <class A, class T, detail::enable_neon_type_t<T> = 0>
+        inline batch<T, A> reduce_max(batch<T, A> const& arg, requires_arch<neon>) noexcept
+        {
+            using register_type = typename batch<T, A>::register_type;
+            const detail::neon_dispatcher::binary dispatcher = {
+                std::make_tuple(wrap::vmaxvq_u8, wrap::vmaxvq_s8, wrap::vmaxvq_u16, wrap::vmaxvq_s16,
```

Comment:
Looks `vmaxvq_xx` requires A64, and there's no `vmaxvq_u64/s64`.
https://developer.arm.com/architectures/instruction-sets/intrinsics/#q=vmaxvq_s8